maquetado/index.html

```diff
@@ -11,11 +11,68 @@
 </head>
 <body>
   <h2 class="title">Front-End Tools</h2>
-  <div class="technology">
-    <img class="logo" src="./assets/react_icon.svg">
-    <h4 class="technology-title">ReactJS</h4>
-    <h4 class="technology-release">Release:2013</h4>
-    <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+  <div class="container">
+    <div class="wrap">
```

Comment:
You could use a more semantic class name for this. For example, if you want the techs to be aligned next to each other you could define a class called `row` a use it in different places.